CMakeLists.txt

```diff
@@ -526,9 +515,27 @@ if(BUSTED_PRG)
     list(APPEND TEST_TARGET_ARGS "USES_TERMINAL")
   endif()
 
-  configure_file(
-    test/config/paths.lua.in
-    ${CMAKE_BINARY_DIR}/test/config/paths.lua)
+  if(NOT (${CMAKE_VERSION} VERSION_LESS 2.8.12))
```

Comment:
`VERSION_GREATER 2.8.11` might be clearer. Or swap the if/else in favor of positive logic.

Reply:
Sure.  I was just following the pattern used a few lines earlier, as suggested by [former you](https://github.com/neovim/neovim/pull/4753#discussion_r63464657). :)

Reply:
Ha! Wow. Well, I intentionally did not suggest the `VERSION_GREATER ... OR ... VERSION_EQUAL` variant.